openrr-apps/src/robot_command.rs

```diff
@@ -98,6 +101,28 @@ pub enum RobotSubCommand {
     Speak { message: Vec<String> },
     /// Execute an external command.
     ExecuteCommand { command: Vec<String> },
+    /// Get navigation current pose.
+    GetNavigationCurrentPose {},
```

Comment:
Is it possible to `GetNavigationCurrentPose {}` -> `GetNavigationCurrentPose`?

It is ok to do it in another change for all members.